Source/WebCore/platform/PublicSuffixStore.h

```diff
@@ -38,14 +38,28 @@ class PublicSuffixStore {
 
     // https://url.spec.whatwg.org/#host-public-suffix
     WEBCORE_EXPORT bool isPublicSuffix(StringView domain) const;
-    WEBCORE_EXPORT String publicSuffix(const URL&) const;
     WEBCORE_EXPORT String topPrivatelyControlledDomain(const String& host) const;
     WEBCORE_EXPORT void clearHostTopPrivatelyControlledDomainCache();
+    WEBCORE_EXPORT void enablePublicSuffixCache();
 
+    class PublicSuffix {
+    public:
+        static PublicSuffix fromRawString(String&& string)
+        {
+            return PublicSuffix(WTFMove(string));
+        }
+        PublicSuffix() = default;
+        bool isValid() const { return !m_string.isEmpty(); }
+        String string() const { return m_string; }
+
+    private:
+        explicit PublicSuffix(const String& string) : m_string(string) { }
```

Comment:
fromRawString() passes a r-value reference but this doesn't and it doesn't WTFMove(), why not?